pkg/expressions/stdlib/funcs.go

```diff
@@ -61,6 +62,14 @@ var StandardFunctions = map[string]KeyBuilderFunction{
 	"tab": kfJoin('\t'),
 	"$":   kfJoin(ArraySeparator),
 
+	// Ranges
+	"$map":    kfArrayMap,
+	"$split":  kfArraySplit,
+	"$join":   kfArrayJoin,
```

Comment:
the use of the `$` symbol leads to confusing issues if not escaped in bash correctly